README.md

```diff
@@ -66,6 +66,18 @@ python setup.py build_sphinx
 
 See the [wiki](https://github.com/SINTEF/PySilCam/wiki) for more details on running PySilCam.
 
+Jupyter notebook setup
+------
+
+After installing, activate the silcam conda environment. Then run:
+
+'''bash
+conda install -c anaconda ipykernel
```

Comment:
> So I wonder if we should install jupyter instead of ipykernel. Jupyter comes with ipykernel, but ipykernel doesn't come with all of jupyter. This is slightly a style decision (as you have to install more with jupyter). But if you are in silcam (on a windows, I assume other OS, but not tested), then you cant type "jupyter notebook" from within the silcam env and have it work (after installing the kernel), cos you only have the kernel, not the jupyter module.
> 
> I note the code does work fine and does what it says it does though.
> 
> And maybe I need a lesson in how to give feedback here... I tried to make a comment to the code.

haha. i am still getting used to this instead of stash. but thanks for the tip - I think your option sounds simpler.